packages/react/src/ActionList/Selection.tsx

```diff
@@ -34,6 +35,15 @@ export const Selection: React.FC<React.PropsWithChildren<SelectionProps>> = ({se
     }
   }
 
+  if (selectionVariant === 'radio') {
+    return (
+      <VisualContainer className={className}>
+        {/* This is just a way to get the visuals from Radio, but it should be ignored in terms of accessibility */}
+        <Radio value="unused" checked={selected} aria-hidden tabIndex={-1} />
```

Comment:
FYI I added the tabIndex -1 here because of an a11y issue that was being reported in the CI checks, lmk if any concerns @hectahertz